pkg/crc/machine/disks.go

```diff
@@ -39,7 +39,12 @@ func ocpGetPVShiftSize(diskSize int, pvSize int) int {
 }
 
 func moveTopolvmPartition(ctx context.Context, shiftSize int, vm *virtualMachine, sshRunner *crcssh.Runner) error {
-	_, _, err := sshRunner.RunPrivileged("move topolvm partition to end of disk", fmt.Sprintf("echo '+%dG,' | sudo sfdisk --move-data /dev/vda -N 5 --force", shiftSize))
+	pvPartition, err := getTopolvmPartition(sshRunner)
+	if err != nil {
+		return err
+	}
+	_, _, err = sshRunner.RunPrivileged("move topolvm partition to end of disk",
+		fmt.Sprintf("echo '+%dG,' | sudo sfdisk --move-data /dev/vda -N %s --force", shiftSize, pvPartition[len("/dev/.da"):]))
```

Comment:
If you use `len("/dev/.da")`, it might be worth also making the `/dev/vda` more generic? Iirc it's `/dev/sda` on Windows?